### DISCUSSION TITLE
BulletDecorator in AvaloniaUI

### DISCUSSION THREAD

**Discussion by Rosentti** [AvaloniaUI/Avalonia#9450] [General]
There is no bulletdecorator class in AvaloniaUI, what should we use instead?

**Comment by maxkatz6** [comment]
[BulletDecorator](https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.primitives.bulletdecorator?view=windowsdesktop-7.0) seems like a pointless control.
What is the issue with typical Panels to organize "bullet" near the content?
And for the list controls, you can do the same in the ItemTemplate.
